samples/client/petstore/c/unit-tests/manual-UserAPI.c

```diff
@@ -102,16 +105,19 @@ int main() {
 		free(loginuserreturn);
 		free(username1);
 		free(password);
+        apiClient_free(apiClient3);
 	}
 
 // logout user test
 	apiClient_t *apiClient4 = apiClient_create();
 
 	UserAPI_logoutUser(apiClient4);
+    apiClient_free(apiClient4);
 
 
 // delete user test
 	apiClient_t *apiClient5 = apiClient_create();
 
 	UserAPI_deleteUser(apiClient5, "example123");
+    apiClient_free(apiClient5);
```

Comment:
Minor suggestion: indent ` apiClient_free` properly with reference to the previous line.

Reply:
You may simply want to use the code formatter to format the manually written test files.

Reply:
I have formatted manually written test files with uncrustify with the configuration we have in auto-generated code.